vanilla-framework/vanilla-framework
# 1522: Align media object metadata items

## Done

- Aligned metadata items regardless of whether there is an icon or not
- Updated the margin between icon and text to match the [design spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Media%20object)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/media-object/media-object/
- Check that the 'CSS frameworks' metadata items is horizontally aligned with the rest of the items that have icons

## Details

Fixes #1522 

## Screenshots

Before:
![screenshot 4](https://user-images.githubusercontent.com/25733845/35327288-297496a4-00f1-11e8-97e3-ecafaf6542ed.png)


After:
![screenshot 5](https://user-images.githubusercontent.com/25733845/35327356-5172452a-00f1-11e8-806d-eb83126fd2e9.png)
